### DISCUSSION TITLE
new_release

### DISCUSSION THREAD

**Discussion by UPTODATE-DEV** [UPTODATE-DEV/devcommunity-frontend#11] [General]
## What's Changed
* Updating app logo & Fix: post content overflow by @UPTODATE-DEV in https://github.com/UPTODATE-DEV/devcommunity-frontend/pull/8
* fix scroll issues by @MauriceBagalwa in https://github.com/UPTODATE-DEV/devcommunity-frontend/pull/9
* Update README.md by @danbaruka in https://github.com/UPTODATE-DEV/devcommunity-frontend/pull/10

## New Contributors
* @UPTODATE-DEV made their first contribution in https://github.com/UPTODATE-DEV/devcommunity-frontend/pull/8
* @MauriceBagalwa made their first contribution in https://github.com/UPTODATE-DEV/devcommunity-frontend/pull/9
* @danbaruka made their first contribution in https://github.com/UPTODATE-DEV/devcommunity-frontend/pull/10

**Full Changelog**: https://github.com/UPTODATE-DEV/devcommunity-frontend/compare/preprod...new_release

<hr /><em>This discussion was created from the release <a href='https://github.com/UPTODATE-DEV/devcommunity-frontend/releases/tag/new_release'>new_release</a>.</em>

**Comment by Rezanmonis** [comment]
Hello

We are looking for frontend and backend developers as our teammates for our edtech project rezanmonis@gmail.com
